### PR TITLE
Implement Pennsylvania CTC contributed reforms

### LINKED ISSUE #7436: Implement Pennsylvania CTC Reform (Contributed)
## Summary

Implement two contributed Pennsylvania Child Tax Credit (CTC) reform proposals for policy analysis.

### Proposal 1: Vermont-Style PA CTC
- Mirrors Vermont's existing CTC structure
- Uses stepped phaseout: 2% reduction rate ($20 per $1,000 increment) above income threshold
- **Key difference from Vermont**: Expand the $125,000 phaseout threshold to ALL filing statuses (Vermont only has a single threshold)
- Age limit and credit amount should be configurable via parameters

### Proposal 2: Federal CTC Match for Young Children
- 50% match of the federal Child Tax Credit
- **Limited to children under age 6 only**
- No income phaseout (simple match structure)

## Technical Requirements

1. **Parameters** in `policyengine_us/parameters/gov/contrib/states/pa/ctc/`
   - Separate folders for each proposal (e.g., `vermont_style/` and `young_child_match/`)
   - Include `in_effect` parameter for each
   - Use active voice for all parameter descriptions

2. **Reform files** in `policyengine_us/reforms/states/pa/ctc/`
   - Create `__init__.py` files
   - Follow existing patterns (e.g., RI CTC, MT CTC)
   - Register in `reforms.py`

3. **Tests** in `policyengine_us/tests/policy/contrib/states/pa/ctc/`
   - Comprehensive test coverage for both proposals
   - Test edge cases: income thresholds, age limits, filing statuses

## Acceptance Criteria

- [ ] Both proposals implemented with full parameter sets
- [ ] `__init__.py` files created for proper imports
- [ ] `reforms.py` updated with imports and registration
- [ ] YAML tests pass for all scenarios
- [ ] `make format` passes
- [ ] All parameter descriptions use active voice

### PR DESCRIPTION
**Pull request by DTrim99**
## Summary

Implements two Pennsylvania Child Tax Credit reform proposals as contributed reforms (not law).

Closes #7436

### 1. Flat Amount CTC (`pa_ctc_flat_amount`)
- **$1,000** per qualifying child under age 6
- **2% phaseout rate** above $125,000 threshold (same for all filing statuses)
- Refundable credit
- Effective 2026

### 2. Federal CTC Match (`pa_ctc_match`)
- **50% match** of the federal Child Tax Credit
- Limited to children under age 6
- No income phaseout
- Refundable credit
- Effective 2026

## Files Added

**Parameters** (`gov/contrib/states/pa/ctc/`):
- `flat_amount/` - in_effect, amount, age_limit, phaseout/threshold, phaseout/rate
- `ctc_match/` - in_effect, match, age_limit

**Reform Code** (`reforms/states/pa/ctc/`):
- `pa_ctc_flat_amount.py`
- `pa_ctc_match.py`
- `__init__.py` files

**Tests** (`tests/policy/contrib/states/pa/ctc/`):
- 8 tests for flat amount reform
- 6 tests for CTC match reform

## Test Plan

- [x] All 14 YAML tests pass
- [x] Tests cover income below/above threshold, phaseout calculations, age limits, filing statuses
- [x] `make format` passes

🤖 Generated with [Claude Code](https://claude.ai/code)

### CODE REVIEW

File: policyengine_us/parameters/gov/contrib/states/pa/ctc/ctc_match/age_limit.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: Pennsylvania provides the CTC match for children younger than this age.
+
+values:
+  2026-01-01: 6
+
+metadata:
+  unit: year
+  period: year
+  label: Pennsylvania CTC match age limit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/contrib/states/pa/ctc/ctc_match/age_limit.yaml]: can you add a reference somewhere to compare against

File: policyengine_us/reforms/states/pa/ctc/pa_ctc_match.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            young_child_ctc = actual_federal_ctc * young_child_share
+
+            # Apply match rate
+            return young_child_ctc * p.match
+
+    def modify_parameters(parameters):
+        # Add pa_ctc_match to refundable credits list
+        refundable = parameters.gov.states.pa.tax.income.credits.refundable
+        current_refundable = refundable(instant("2026-01-01"))
+        if "pa_ctc_match" not in current_refundable:
```
Comment (PavelMakarchuk) [inline policyengine_us/reforms/states/pa/ctc/pa_ctc_match.py:69]: I believe both reforms can not be applies at the same time, is that expected?

File: policyengine_us/tests/policy/contrib/states/pa/ctc/pa_ctc_flat_amount.yaml [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+      tax_unit:
+        members: [person1, child1]
+    households:
+      household:
+        members: [person1, child1]
+        state_code: PA
+  output:
+    # AGI $100k < threshold $125k, no phaseout
+    # Credit = 1 child * $1,000 = $1,000
+    pa_ctc_flat_amount: 1_000
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/contrib/states/pa/ctc/pa_ctc_flat_amount.yaml:28]: can you add net income and the pa_income_tax as outputs to the tests. Can we make sure those flows through all the way through the computation tree.

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/reforms/states/pa/ctc/pa_ctc_match.py (hunk 2 above)
Comment (DTrim99) [reply policyengine_us/reforms/states/pa/ctc/pa_ctc_match.py:69]: Yes, only one is active at a time.

File: policyengine_us/parameters/gov/contrib/states/pa/ctc/ctc_match/age_limit.yaml (hunk 1 above)
Comment (DTrim99) [reply policyengine_us/parameters/gov/contrib/states/pa/ctc/ctc_match/age_limit.yaml]: This is a contributed reform from advocates; there are no references.
